apps/communications/dialer/test/marionette/keypad_test.js

```diff
@@ -7,7 +7,8 @@ var ReflowHelper =
 marionette('Dialer > Keypad', function() {
   var assert = require('assert');
 
-  var client = marionette.client({ profile: Dialer.config });
+  var client = marionette.client({ profile: Dialer.config,
+  desiredCapabilities: { raisesAccessibilityExceptions: false } });
```

Comment:
Johan, I really do not understand this :( if I remove that line, or add it inside the dialer.js
Dialer.config = {
  settings: {
    'devtools.overlay': true,
    'hud.reflows': true
  },
  prefs: {
    'devtools.debugger.forbid-certified-apps': false
  },
desiredCapabilities: { raisesAccessibilityExceptions: false }
};
The test fails... lementNotAccessibleError: ElementNotAccessibleError: Element is not focusable via the accessibility API -> id: , tagName: DIV, className: keypad-key